llvm/test/CodeGen/X86/fast-isel-nontemporal.ll

```diff
@@ -94,14 +94,15 @@ entry:
 
 ;
 ; MMX Store
+; Note: doesn't actually emit a non-temporal store here.
 ;
 
 define void @test_mmx(ptr nocapture %a0, ptr nocapture %a1) {
 ; ALL-LABEL: test_mmx:
 ; ALL:       # %bb.0: # %entry
 ; ALL-NEXT:    movq (%rdi), %mm0
 ; ALL-NEXT:    psrlq $3, %mm0
-; ALL-NEXT:    movntq %mm0, (%rsi)
+; ALL-NEXT:    movq %mm0, (%rsi)
```

Comment:
The `nontemporal` not respected?

Reply:
Correct; this already wasn't happening under SDag ISel. Now it _also_ isn't happening under fast ISel.

Reply:
@RKSimon's 25952ec64791e781f9bdd7bf29ab78f0095b8ee7 fixed this.